180atp/apibodyexamples/query_by_GET.txt

```diff
@@ -0,0 +1 @@
+https://inventoryservice.{RegionShortName}-il301.gateway.prod.island.powerapps.com/api/environment/{EnvironmentId}/onhand?organizationId=usmf&productId=Bike&SiteId=1&LocationId=11&groupBy=ColorId,SizeId&returnNegative=true&QueryATP=true
```

Comment:
Remove il301 as this may not be the case for all users

Reply:
removed ：）